### PR TITLE
New App Design

### PR DESCRIPTION
**Pull request by cleptric**
A new design for the app template.

Screenshot 2019-05-21 at 20 59 40
Screenshot 2019-05-21 at 20 59 55
Screenshot 2019-05-21 at 21 00 00
Screenshot 2019-05-21 at 21 01 01

The css is based on custom build of milligram, to fix some bugs and adopt the CakePHP CI.
Related PR for bake is cakephp/bake#550.

A missing task is to find a better approach for the related actions.
I don't want to clutter the whole sidebar with actions to related models, so I'm thinking to go into a menu direction for the models only.

### CODE REVIEW

**Comment by ADmad** [comment]
Love the updated look and fact that milligram allows for creating clean HTML with minimal extra tags and css classes.

My only concern is milligram is a one man project and doesn't seems to be actively maintained currently. It's last release was over 2 years ago https://github.com/milligram/milligram/releases.

**Comment by cleptric** [comment]
@ADmad That's a valid concern. I checked other mini css frameworks, most of them haven't seen a release for years as well or just need to much work to make look as good as milligram.
The actual amount of sass is very little, so it's pretty easy to change things around.
The obvious bugs I found are already fixed.
I may consider to continue tweaking it and release it as an updated version on my own.

**Comment by cleptric** [comment]
Screenshot 2019-05-21 at 23 01 44
New style for DB text columns

**Comment by inoas** [comment]
All nice and such but what would really improve day-to-day developer experience is classless or just bootstrap-4-grid/responsive utilities classes in cake bake templates and basic app+error layouts instead of the current tag and class soup

**Comment by cleptric** [comment]
@inoas Please have a look at the updated bake templates. The is barely any calls nor tag soup left.
Adding different css should be a task of a few minutes now compared to before.

**Comment by inoas** [comment]
aside the double pagiantor/pagination class this looks really nice now!
"button float-right" seems a little unnecessary. Also I'd go with either always buttons or only for "write" operations

**Comment by ADmad** [comment]
Good to know there's already another framework using milligram. That improves the chance of milligram being maintained in future.

File: templates/layout/default.php [hunk 1]
```diff
@@ -11,7 +11,6 @@
  * @link          https://cakephp.org CakePHP(tm) Project
  * @since         0.10.0
  * @license       https://opensource.org/licenses/mit-license.php MIT License
- * @var \App\View\AppView $this
```
Comment (dereuromark) [inline templates/layout/default.php]: Those need to be recovered!

**Comment by bravo-kernel** [comment]
❤️

File: templates/layout/default.php (hunk 1 above)
Comment (cleptric) [reply templates/layout/default.php]: Will follow up on this

**Pull request merged**